packages/docs/docs/guide/deploy.md

```diff
@@ -185,34 +185,27 @@ heroku open
 
 1. Install the Now CLI globally: `npm install -g now`
 
-2. Add a `docs.now.json` file to your project root:
+2. Add a `now.json` file to your project root:
 
     ```json
     {
-      "name": "my-cool-docs",
-      "type": "static",
-      "static": {
-        "public": "docs/.vuepress/dist"
-      },
-      "alias": "my-cool-docs",
-      "files": [
-        "docs/.vuepress/dist"
-      ]
-    }
+        "version": 2,
+        "name": "vuepress",
+        "alias": "vuepress.now.sh",
+        "builds": [{
+          "src": "package.json",
+          "use": "@now/static-build"
+        }]
+      }
```

Comment:
Maybe the indent is inconsistent?

Reply:
Thanks for the feedback. Format resolved.